webextensions/api/browsingData.json

```
@@ -693,7 +693,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/settings",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "27"
```

:x: https://github.com/chromium/chromium/commit/c606b46a266fcef9370b02a2db8f08c8bc57fbcf#diff-a800f42aae59d230bd825829bdca353c1fdb9152b89234803d53033afe0a9be5R118 landed in [26.0.1404.0](https://github.com/chromium/chromium/releases/tag/26.0.1404.0) before Chrome 26.0.1410.

```suggestion
                "version_added": "26"
```